src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs

```diff
@@ -22,21 +22,24 @@ public class SystemLogger : ILogger
         private readonly LogLevel _logLevel;
         private readonly IDebugStateProvider _debugStateProvider;
         private readonly IScriptEventManager _eventManager;
+        private readonly IExternalScopeProvider _scopeProvider;
```

Comment:
All changes dealing with `IExternalScopeProvider` are fairly boilerplate. They're removing `DictionaryLoggerScope` and going with the built-in scope provider (which didn't exist when we first built this). It will clean up all the `AsyncLocal` stuff we were doing and consolidate it into one provider.